ballerina/tests/xml_from_json_test.bal

```diff
@@ -21,7 +21,7 @@ import ballerina/test;
 }
 isolated function testJsonDataSize() returns error? {
     json data = {id: 30};
-    xml expected = xml `<id>30</id>`;
+    xml expected = xml `<root><id>30</id></root>`;
     xml? result = check fromJson(data);
```

Comment:
will this change the behavior, why we need to change the output here

Reply:
Yes, this will change the output behavior. 
Earlier, when the converted XML has multiple elements, the root tag has been added to get a valid XML. 
Eg:
JSON data:
```
{
 genre: "Sci-Fi",
 language: "German"
}
```
XML Output for given JSON:
```
<genre>Sci-Fi</genre>
<language>German</language>
```
This is not a valid XML. So, created a root tag.

But now, if the output XML is not empty or text, the root tag will be created.